app/models/site_status.rb

```diff
@@ -43,4 +49,20 @@ class SiteStatus < ApplicationRecord
   }
   scope :with_vacancies, -> { where.not(vac_status: :no_vacancies) }
   scope :open_for_applications, -> { findable.applications_being_accepted_now.with_vacancies }
+
+private
+
+  def vac_status_consistent_with_course_study_mode?
+    if no_vacancies?
```

Comment:
A case statement might be easier to read...

Reply:
The linter objects to case statements without a subject, unfortunately.

In any case, we think we're gonna back out the actual validation from this PR and just take the spec changes forward, so #196 will have the actual code.

Reply:
@benilovj could you just use `case vac_status: no_vacancies` instead?

Reply:
Dunno if it's clear, but I think @dankmitchell means:

```
case vac_status
when 'no_vacancies'
  ...
```

Reply:
Thanks @misaka 💯

Reply:
Fixed.